### PR TITLE
Remove `Universal Lemma Under Conjunction` flag (deprecated in 8.15).

### PR DESCRIPTION
**Pull request by Zimmi48**
- [x] Added / updated **test-suite**.

- [x] Added **changelog**.
- [x] Added / updated **documentation**.

### CODE REVIEW

**Review by ppedrot — APPROVED** [review]

File: doc/changelog/07-vernac-commands-and-options/15268-test-remove-univ-lemma-under-conj.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+- **Removed:**
+  `Universal Lemma Under Conjunction` flag, that was deprecated in 8.15
```
Comment (jfehrle) [inline doc/changelog/07-vernac-commands-and-options/15268-test-remove-univ-lemma-under-conj.rst:2]:
```suggestion
  `Universal Lemma Under Conjunction` flag that was deprecated in 8.15
```

**Review by jfehrle — APPROVED** [review]
LGTM.

**Comment by ppedrot** [comment]
@Zimmi48 do you want to tweak the changelog before I merge?

**Comment by Zimmi48** [comment]
I'm not convinced by the suggested change.

**Comment by ppedrot** [comment]
OK, let's merge then.

**Comment by ppedrot** [comment]
@coqbot merge now

**Pull request merged**